docs/cpp/guides/status.md

```diff
@@ -158,7 +158,7 @@ if (!s.ok()) {  // Either Open or Create failed
 }
 ```
 
-### Returning a Status or a Value
```

Comment:
I think this is intended to be a sub-section of "Using Status for Returning Errors". Same below.

Reply:
cl/460480569 makes it look like this is an intentional choice. Quite a bit of stuff is stripped out and it seems to make more sense when this other stuff is still in there.

Reply:
SGTM.